frontend/src/components/cards/MiniCardComponent.js

```diff
@@ -13,7 +13,7 @@ const useStyles = createUseStyles((theme) => ({
         // marginTop:50,
         // padding: '8px 16px 8px 16px',
         '&:hover': {
-            borderColor: theme.uniformStyle.color.tableHeaderColor,
+            borderColor: '#AE56FF',
```

Comment:
please add at least a comment here, or maybe consider creating a variable for this color, because like this its a "Magic value" 🙂

Reply:
I am on it 😄